packages/account/src/interface.cairo

```diff
@@ -14,37 +14,71 @@ pub const ISRC6_ID: felt252 = 0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c60
 
 #[starknet::interface]
 pub trait ISRC6<TState> {
+    /// Executes a list of calls from the account.
     fn __execute__(self: @TState, calls: Array<Call>) -> Array<Span<felt252>>;
+
+    /// Verifies the validity of the signature for the current transaction.
+    /// This function is used by the protocol to verify `invoke` transactions.
     fn __validate__(self: @TState, calls: Array<Call>) -> felt252;
+
+    /// Verifies that the given signature is valid for the given hash.
     fn is_valid_signature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
 }
 
 #[starknet::interface]
 pub trait IDeclarer<TState> {
+    /// Verifies the validity of the signature for the current transaction.
+    /// This function is used by the protocol to verify `declare` transactions.
     fn __validate_declare__(self: @TState, class_hash: felt252) -> felt252;
 }
 
 #[starknet::interface]
 pub trait IDeployable<TState> {
+    /// Verifies the validity of the signature for the current transaction.
+    /// This function is used by the protocol to verify `deploy_account` transactions.
     fn __validate_deploy__(
         self: @TState, class_hash: felt252, contract_address_salt: felt252, public_key: felt252,
     ) -> felt252;
 }
 
 #[starknet::interface]
 pub trait IPublicKey<TState> {
+    /// Returns the current public key of the account.
     fn get_public_key(self: @TState) -> felt252;
+
+    /// Sets the public key of the account to `new_public_key`.
+    ///
+    /// Requirements:
+    ///
+    /// - The caller must be the contract itself.
+    /// - The signature must be valid for the new owner.
+    ///
+    /// Emits both an `OwnerRemoved` and an `OwnerAdded` event.
     fn set_public_key(ref self: TState, new_public_key: felt252, signature: Span<felt252>);
 }
 
 #[starknet::interface]
 pub trait ISRC6CamelOnly<TState> {
+    /// Verifies that the given signature is valid for the given hash.
+    /// This is a camelCase version of `is_valid_signature` for compatibility.
     fn isValidSignature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
 }
```

Comment:
```suggestion
/// Adds camelCase support for `ISRC6`.
#[starknet::interface]
pub trait ISRC6CamelOnly<TState> {
    fn isValidSignature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
}
```

For camelCase interfaces let's use this pattern to avoid repeating the comments.